pkgs/development/libraries/libusbsio/makefile.patch

```diff
@@ -0,0 +1,23 @@
+diff -u a/makefile b/makefile
+--- a/makefile	2023-11-15 00:58:47.844331932 +1100
++++ b/makefile	2023-11-15 01:02:29.176079911 +1100
+@@ -6,8 +6,8 @@
+ #  NXP USBSIO Library Makefile
+ # 
+ 
+-CC = gcc

```

Comment:
Wouldn't setting `"CC=${stdenv.cc.targetPrefix}cc"` in `makeFlags` be enough?

Reply:
Thanks for the suggestion, I didn't know about `makeFlags`. I will try it out

Reply:
Thanks again @wegank, using your suggestion I removed the patch.